src/ChannelServer/Util/GmCommands.Handlers.cs

```diff
@@ -519,6 +519,7 @@ private CommandResult HandleSpeed(ChannelConnection conn, Character sender, Char
 
 			target.Speed = speed;
 			Send.ZC_MOVE_SPEED(target);
+			Send.ZC_OBJECT_PROPERTY(target, PropertyId.PC.MSPD);
```

Comment:
Does this make the line above redundant?

Reply:
This is for the F1 window so that you can see the changed Speed. "Movement Speed: XX"

Reply:
So it doesn't actually change the movement speed? Alright.